tests/test_invenio_communities.py

```diff
@@ -284,34 +284,39 @@ def assert_header_links(response, ref, page, size):
 
 def test_communities_rest_get_details(app, db, communities):
     """Test the OAI-PMH Sets creation."""
+    (comm1, comm2, comm3) = communities
     with app.test_client() as client:
-        response = client.get('/api/communities/comm1')
+        response = client.get('/api/communities/{}'.format(str(comm1.id)))
```

Comment:
only the API yes. As the UUIDs are written inside the records, one needs to retrieve the community info using the UUID.
It's also possible to allow retrieving by either name or UUID as both are unique. We would just need to make sure that names which look like UUIDs are not allowed. Is the communities API part of the API currently documented on Zenodo? If yes then we will need to keep it backward compatible.